src/main/java/com/gd/clinic/security/service/UserService.java

```diff
@@ -0,0 +1,29 @@
+package com.gd.clinic.security.service;
+
+import com.gd.clinic.security.entity.User;
+import com.gd.clinic.security.repository.UserRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.Optional;
+
+@Service
+@Transactional
+public class UserService {
```

Comment:
please apply my comments from another classes - `@RequiredArgsConstructor`, empty lines and return value for `save` method